static/locales/es.json

```
@@ -1577,8 +1579,8 @@
             "vllm_omni": "vLLM-Omni (Clon local)"
         },
         "sourcePreset": "Predefinido",
-        "sourceClone": "Clonado",
-        "sourceDesign": "Generado por descripción",
+        "sourceClone": "Clonación de voz",
```

Biome (2.5.1)

[error] The key sourceClone was already declared.

(lint/suspicious/noDuplicateObjectKeys)